### PR TITLE
Update: issue templates

### PR DESCRIPTION
**Pull request by dnikub**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+about: Tell us about a bug or accessibility issue you may have identified.
+title: Provide a general summary of the issue
+labels: bug
+assignees: ''
+
+---
+
+If you think you've found a bug or accessibility issue, feel free to let us know. Before you continue filling out this issue, please answer this checklist for yourself:
+
+1. I ensured that the problem is not just caused by a simple error in my own console
```
Comment (dnikub) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
1. I ensured that the problem is not caused by my local environment (e.g., a browser plugin, a really old browser version or unusual browser settings)
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+assignees: ''
+
+---
+
+If you think you've found a bug or accessibility issue, feel free to let us know. Before you continue filling out this issue, please answer this checklist for yourself:
+
+1. I ensured that the problem is not caused by my local environment (e.g., a browser plugin, a really old browser version or unusual browser settings)
+2. I checked that no GitHub issue was already filed for this bug
+
+If you answered everything with yes, go ahead and with this GitHub issue. Be sure to include as much information as possible so we can reproduce the bug.
```
Comment (thomasdax98) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
If you answered everything with yes, go ahead with this GitHub issue. Be sure to include as much information as possible so we can reproduce the bug.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+about: Suggest an idea for a11yphant
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+Feature requests are very welcome. But take a moment to find out whether your idea fits with the scope and aims of the project. It's up to you to make a strong case to convince us of this feature. Please provide as much detail and context as possible.
+
+**Please discuss your proposed changes with us before starting implementation. Otherwise, they might collide with our plans.**
```
Comment (thomasdax98) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Should we remove this here too? Or leave it as a warning not to implement anything?

Comment (dnikub) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
**Please don't submit any custom implementations. We currently do not accept pull requests. However, you are welcome to submit ideas for new features.**
```

Comment (dnikub) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Maybe like this @thomasdax98?

Comment (thomasdax98) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Yes, that's okay for me 👍

**Review by thomasdax98 — APPROVED** [review]

**Review by lucapircher — APPROVED** [review]

**Pull request merged**